container-system.nix

```diff
@@ -1,7 +1,13 @@
+{
+  pkgs ? import <nixpkgs> {},
+  lib ? pkgs.lib,
+  # older version of nixos-container is required for compatibility with older systemd
+  nixos-container ? (import (builtins.fetchTarball https://github.com/NixOS/nixpkgs-channels/archive/nixos-19.09.tar.gz) {}).nixos-container
```

Comment:
perhaps it would make sense to remove the default here and move the comment into the flake?

Reply:
I would prefer it to work without flakes as it did before, but I guess we can also set it up to work via flake-compat.

Reply:
That makes sense to me.  I just haven't used flakes much...